docs/spark-qualification-tool.md

```diff
@@ -313,95 +338,183 @@ $SPARK_HOME/bin/spark-shell --jars rapids-4-spark-tools_2.12-<version>.jar
-10. SQL Duration with Potential Problems : Time duration of any SQL/DF operations that contains 
-    operations we consider potentially problematic. 
-11. SQL Ids with Failures: SQL Ids of queries with failed jobs.
-12. Read Score Percent: The value of the parameter `--read-score-percent` when the Qualification tool was run. Default is 20 percent. 
-13. Read File Format Score: A score given based on whether the read file formats and types are supported.
-14. Unsupported Read File Formats and Types: Looks at the Read Schema and
-    reports the file formats along with types which may not be fully supported.
-    Example: Parquet[decimal], JDBC[*]. Note that this is based on the current version of the plugin and
-    future versions may add support for more file formats and types.
-15. Unsupported Write Data Format: Reports the data format which we currently don’t support, i.e.
-    if the result is written in JSON or CSV format.
-16. Complex Types: Looks at the Read Schema and reports if there are any complex types(array, struct or maps) in the schema.
-17. Nested Complex Types: Nested complex types are complex types which
-    contain other complex types (Example: `array<struct<string,string>>`). 
-    Note that it can read all the schemas for DataSource V1. The Data Source V2 truncates the schema,
-    so if you see ..., then the full schema is not available.
-    For such schemas we read until ... and report if there are any complex types and nested complex types in that.
-
-Note that SQL queries that contain failed jobs are not included.
-
-## Qualification tool score algorithm
-
-In the Qualification tool’s output, all applications are ranked based on a “score”. 
-
-The score is based on the total time spent in tasks of SQL Dataframe operations.
-The tool also looks for read data formats and types that the plugin doesn't fully support and if it finds any,
-it will take away from the score. The parameter to control this negative impact of the
-score is  `-r, --read-score-percent` with the default value as 20(percent).
-Each application(event log) could have multiple SQL queries. If a SQL's plan has a Dataset API or RDD call
-inside of it, that SQL query is not categorized as a Dataframe SQL query. We are unable to determine how much
-of that query is made up of Dataset or RDD calls so the entire query task time is not included in the score.
-
-The idea behind this algorithm is that the longer the total task time doing SQL Dataframe operations
-the higher the score is and the more likely the plugin will be able to help accelerate that application.
-
-Note: The score does not guarantee there will be GPU acceleration and we are continuously improving
-the score algorithm to qualify the best queries for GPU acceleration.
+### HTML Report
+
+Starting from Version-22.06, the Qualification tool generates HTML formatted report. The HTML report can be disabled by
```

Comment:
we might want to put this section first so people see html output before anything and hopefully use it

Reply:
Done!